src/utils/urls.js

```diff
@@ -1,5 +1,12 @@
+function check() {
+  if (process.env.REACT_APP_LOCAL_ENV === 'true') {
+    return 'http://' + window.location.hostname + ':4000';
```

Comment:
Wouldn't writing `0.0.0.0:4000` be the same?

Reply:
Writing 0.0.0.0:4000 doesn't work as its not relative and the JS tries to connect to a server running on the computer in which its running